src/Dapper.Bulk.Shared/PropertiesCache.cs

```diff
@@ -102,7 +102,8 @@ public static List<PropertyInfo> ComputedPropertiesCache(Type type)
             foreach (var prop in props)
             {
                 var columnAttr = prop.GetCustomAttributes(false).SingleOrDefault(attr => attr.GetType().Name == "ColumnAttribute") as dynamic;
```

Comment:
perhaps it would be better to write
`attr.GetType() == typeof(ColumnAttribute)`
but it may cause major changes in code